experimental/PropertyDDS/packages/property-binder/tsconfig.json

```diff
@@ -6,6 +6,8 @@
 		"allowJs": true,
 		"outDir": "./lib",
 		"rootDir": "./src",
+		// Targeting ES2022 causes registerOnPath to not cause callbacks for unknown reasons, so avoid updating.
```

Comment:
This is pretty creepy, but it has excellent test courage (a ton of tests fail), so this workaround seems safe. If we go ahead with this PR, we might want to have one of the PropertyDDS devs take a look at it.